org.filmulator.Filmulator.json

```diff
@@ -0,0 +1,178 @@
+{
+    "app-id": "org.filmulator.Filmulator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "filmulator-gui",
+    "rename-desktop-file": "filmulator-gui.desktop",
+    "rename-icon": "filmulator64icon",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
```

Comment:
you don't actually need that share when doing fallback-x11. it's included with that permission (separating it in x11 was a mistake but... backwards compatibility!).

Reply:
good to know.